docs/user/overview.md

```diff
@@ -60,7 +60,7 @@ mv initial/install-config.yaml .
 rm -rf initial
```

Comment:
@crawford @wking what is the name of the "hidden file" mentioned above? It is quite mysterious to me.

Reply:
`.openshift_install_state.json`